compiler-plugin/src/main/java/io/ballerina/c2c/utils/DockerGenerator.java

```diff
@@ -227,12 +261,122 @@ private String entryPointArgBuilder(List<String> args) {
                 "[" + String.join(",", args.stream().map(s -> "\"" + s + "\"").toArray(String[]::new)) + "]";
     }
 
+    private void addInitialDockerContent(StringBuilder dockerfileContent) {
+        dockerfileContent.append("# Auto Generated Dockerfile").append(LINE_SEPARATOR);
+        dockerfileContent.append("FROM ").append(this.dockerModel.getBaseImage()).append(LINE_SEPARATOR);
+        dockerfileContent.append(LINE_SEPARATOR);
+        dockerfileContent.append("LABEL maintainer=\"dev@ballerina.io\"").append(LINE_SEPARATOR);
+    }
+
+    private String generateTestDockerFile(Path testSuiteJsonPath, Path jacocoAgentJarPath) throws DockerGenException {
+        StringBuilder testDockerFileContent = new StringBuilder();
+        addInitialDockerContent(testDockerFileContent);
+
+        //copy the test suite json
+        testDockerFileContent.append("COPY ");
+        Optional<Path> testSuiteJsonPathOptional = Optional.ofNullable(testSuiteJsonPath.getFileName());
+        if (testSuiteJsonPathOptional.isPresent()) {
+            testDockerFileContent.append(testSuiteJsonPathOptional.get())
+                    .append(" ").append(getTestSuiteJsonCopiedDir()).append("/ ").append(LINE_SEPARATOR);
+        } else {
+            throw new DockerGenException("Test suite json path is not provided");
+        }
+        new TreeSet<>(this.dockerModel.getDependencyJarPaths())
+                .stream()
+                .map(Path::getFileName)
+                .forEach(path -> {
+                    testDockerFileContent.append("COPY ")
+                                    .append(path)
+                                    .append(" ").append(getWorkDir())
+                                    .append("/jars/ ").append(LINE_SEPARATOR);
+                            //TODO: Remove once https://github.com/moby/moby/issues/37965 is fixed.
+                            boolean isCiBuild = "true".equals(System.getenv().get("CI_BUILD"));
```

Comment:
We can remove this `CI_BUILD` configs now

Reply:
> We can remove this `CI_BUILD` configs now

Shall I remove from both places? Line no:206 and 292

Reply:
Done in [5fc44be](https://github.com/ballerina-platform/module-ballerina-c2c/pull/765/commits/5fc44befe4d898c702ef885cea29fd93df2f73cb)